### ISSUE TITLE
Not all session proposal content is visible

### ISSUE THREAD

**Issue by chrisjaure** [BarCampGreenBay/barcampgb.org#43]
The index card metaphor is pretty strict and doesn't allow for multi-line titles or text content that's too long. Need a way to make it more flexible or show all info.

**Comment by Perrishnikov** [comment]
Looks fine to me

**Comment by justinfriebel** [comment]
I could update the layout to be something like this:

![image](https://cloud.githubusercontent.com/assets/4109296/19131286/83f6f170-8b14-11e6-95e3-aa8863f7850a.png)

**Comment by chrisjaure** [comment]
@justinfriebel you propose removing the background lines and letting the text flow naturally?

**Comment by justinfriebel** [comment]
@chrisjaure yes. This photo also shows the cards using a fluid container as to fill the whole width on larger screens.

**Comment by chrisjaure** [comment]
I'd like to keep the notecard look and feel if possible. Also keep in mind mobile so optimizing for desktop only won't completely solve the problem.
